server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -57,6 +57,30 @@ private FieldMasks() {
     }
 
     /**
+     * Creates new instance of {@code FieldMask} from given type {@link Descriptors.Descriptor descriptor} and
+     * protobuf field numbers.
+     *
+     * @param typeDescriptor {@link Descriptors.Descriptor descriptor} of the type to create a mask for.
+     * @param fieldNumbers   numbers of fields to include in the mask as they are defined in a corresponding
+     *                       {@code .proto} file.
+     * @return new {@code FieldMask} of given type for mentioned fields.
+     */
+    public static FieldMask maskOf(Descriptors.Descriptor typeDescriptor, int... fieldNumbers) {
```

Comment:
@armiol I used this naming because of method name `Descriptor#findFieldByNumber`